examples/check-downloaded-file-name-and-content/README.MD

```diff
@@ -0,0 +1,21 @@
+# Check the Downloaded File Name and Content
+
+**Test Code**: [index.js](index.js)
+
+This example shows how to check the name and the content of a file downloaded during a test.
+
+During the test, TestCafe clicks the link that triggers file download. To confirm download success and obtain the file name and content, a [RequestLogger](https://devexpress.github.io/testcafe/documentation/reference/test-api/requestlogger/) is attached to the test with the [t.requestHooks](https://devexpress.github.io/testcafe/documentation/reference/test-api/test/requesthooks.html) method.
+
+The [RequestLogger.contains](https://devexpress.github.io/testcafe/documentation/reference/test-api/requestlogger/contains.html) method in combination with the [t.expect.ok](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/expect/ok.html) verify the downloaded file name and content.
+
+The Request Logger is set up to log all successful `GET` requests to a particular URL. The responses are then filtered to only contain a `.txt` file with specified contents.
+
+The Request Logger has the following [options](https://devexpress.github.io/testcafe/documentation/reference/test-api/requestlogger/constructor.html) enabled:
+
+|Option|Effect|
+|-|-|
+|`logResponseHeaders`| logs the response header|
+|`logResponseBody`| logs the response body|
+|`stringifyResponseBody`| converts the response body to a string|
+
+The file name is read from the `content-disposition` response header. The file contents constitute the response body.
```

Comment:
`The file contents constitute the response body.`

Can you clarify what you mean by "constitute"? For example, can we say:

`The file contents contain the response body.`

Reply:
I meant to say that "the response body is the file contents". 
If the response body is literally the only thing inside the file, can we still say `The file contents contain the response body` ?

Reply:
Thank you. Please change it to:

`The response body contains the contents of the file.`

Reply:
> The response body is added to the file content.